src/constructs/aws/Queue.ts

```diff
@@ -251,6 +252,10 @@ export class Queue extends AwsConstruct {
             description: `URL of the "${id}" SQS queue.`,
             value: this.queue.queueUrl,
         });
+        this.dlqArnOutput = new CfnOutput(this, "DlqArn", {
+            description: `ARN of the "${id}" SQS dead letter queue.`,
+            value: this.dlq.queueArn,
+        });
```

Comment:
This CloudFormation output is not needed as far as I know.

Reply:
Right, but the `${construct:jobs.dlqArn}` is transformed into a `!Ref` (cloudformation reference), it's not using the CF output.

The variables are exposed in the `variables()` method (not via CF outputs -> these are used for the CLI output).

Reply:
Ah - yes my bad, I believed it was the other way around. I've pushed a change that removes the initializer and the dlqArnOutput prop